src/test/java/uk/gov/pay/ledger/agreement/AgreementDaoIT.java

```diff
@@ -91,8 +90,8 @@ void shouldFindAndGuaranteeMostRecentlyCreatedPaymentInstrumentByExternalId() {
 
     @Test
     void shouldSearchAgreement() {
-        var fixture = AgreementFixture.anAgreementFixture("external-id", "service-id");
-        var secondFixture = AgreementFixture.anAgreementFixture("second-external-id", "second-service-id");
+        var fixture = AgreementFixture.anAgreementFixture().withExternalId("external-id").withServiceId("service-id");
```

Comment:
If we want to be consistent with the formatting of other places, we should use each `withXXXXX` on a new line

Reply:
updated